python/docs/source/tutorials/python_Contextual_bandits_and_Vowpal_Wabbit.ipynb

```diff
@@ -405,43 +403,45 @@
     "\n",
     "  choice = vw.predict(test_example)\n",
     "  print(j, choice)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     ">**Note:** The contextual bandit assigns every instance to the third action as it should per the cost structure of the train data. You can save and load the model you train from a file.\n",
     "\n",
     "Finally, experiment with the cost structure to see that the contextual bandit updates its predictions accordingly."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "vw.save('cb.model')\n",
     "del vw\n",
     "\n",
     "vw = pyvw.vw(\"--cb 4 -i cb.model\")\n",
     "print(vw.predict('| a b'))"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "The `-i` argument means input regressor, telling Vowpal Wabbit to load a model from that file instead of starting from scratch.\n",
     "\n",
     "## More to explore\n",
     "\n",
-    "- Review the [example Python notebooks](https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/python/examples).\n",
+    "- Review the [example Python notebooks](https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/python/docs/source/examples).\n",
```

Comment:
Maybe this link is a better location to link to? https://vowpalwabbit.org/docs/vowpal_wabbit/python/latest/examples/

Reply:
I can definitely do that. That's an even better place to point to.